### PR TITLE
Python: Model `flask.jsonify`

### PR DESCRIPTION
**Pull request by RasmusWL**

### CODE REVIEW

File: python/ql/lib/semmle/python/frameworks/Flask.qll [hunk 1]
```diff
@@ -171,6 +171,24 @@ module Flask {
...
+     *
+     * See
+     * - https://flask.palletsprojects.com/en/2.2.x/api/#flask.json.jsonify
+     */
+    private class FlaskJsonifyCall extends InstanceSource, DataFlow::CallCfgNode {
+      FlaskJsonifyCall() {
+        this = API::moduleImport("flask").getMember("jsonify").getACall()
+      }
+
+      override DataFlow::Node getBody() { result = this.getArg(0) }
```
Comment (tausbn) [inline python/ql/lib/semmle/python/frameworks/Flask.qll:185]: Looking at the documentation for `flask.jsonify`, it seems like it's really _any_ argument that's put into the response, so maybe this should be `getArg(_)` (or something similar)?

**Review by tausbn — APPROVED** [review]
One minor question, otherwise this looks good to me. :+1:

File: python/ql/lib/semmle/python/frameworks/Flask.qll (hunk 1 above)
Comment (RasmusWL) [reply python/ql/lib/semmle/python/frameworks/Flask.qll]: good catch :+1:

**Review by tausbn — APPROVED** [review]
Nice!

**Pull request merged**